tests/fixtures/Address.json

```diff
@@ -0,0 +1,17 @@
+[
+{
+    "model": "registry.address",
+    "pk": "3618160d-cd9e-4b9e-b535-a9dfda6b6e3e",
+    "fields": {
+        "address_line_1": "657 Baker Motorway",
+        "address_line_2": "Christinafurt, MS 91987",
+        "address_line_3": null,
+        "postcode": "66589",
+        "city": "Harrisonfurt",
+        "state": "Wisconsin",
+        "country": "IN",
```

Comment:
Recommend making it "Indian" with state and city. For e..g see: https://github.com/openskies-sh/aerobridge/blob/master/fixtures/initial_data.json

Reply:
Done in 2ef8fa8c3fcff67b6cafd105b39529cf5107983c